### PR TITLE
[bug] [doc] fix  the data quality link error about the navigation of dolphinscheduler 3.0.0-alpha zh docs 

### PR DESCRIPTION
**Pull request by chens616**
https://github.com/apache/dolphinscheduler/issues/9886

## Purpose of the pull request

This pull request fixs the data quality link about the navigation of dolphinscheduler 3.0.0-alpha zh docs  .

## Brief change log

I remove a space on data quality link of the navigation file of dolphinscheduler 3.0.0-alpha zh docs  .

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: et --soft [hunk 1]
```diff
@@ -0,0 +1,242 @@
+[33mcommit 404f7c8b06c19a3db3b25e161f495f795cdef81b[m[33m ([m[1;36mHEAD -> [m[1;32mdev[m[33m, [m[1;31morigin/dev[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```
Comment (SbloodyS) [inline et --soft]: This file should not be included in this PR.

**Review by SbloodyS — CHANGES_REQUESTED** [review]

**Review by SbloodyS — APPROVED** [review]
LGTM. PTAL @zhongjiajie

**Review by zhuangchong — APPROVED** [review]

**Comment by zhuangchong** [comment]
@chens616 
Thank you very much for your contribution, this PR is very good. I think we can get deep communiction, you can contact me through mail or add wechat(KerwinZhuang), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.
